DEV-GUIDE.md

```diff
@@ -287,7 +288,7 @@ make test
 You can get the current unit test coverage numbers on your local repo by going to the top of the repo and entering:
 
 ```shell
-make coverage
+make cov
```

Comment:
alternatively, could you rename the cov target to coverage? More readable..

Reply:
sure - I've submitted PR: https://github.com/istio/istio/pull/2394.  Cheers.